project_templates/conda_exercise/exercise_1/pear.xml

```diff
@@ -0,0 +1,165 @@
+<tool id="iuc_pear" name="Pear" version="0.9.6.1">
+    <description>Paired-End read merger</description>
+    <stdio>
+        <exit_code range="1:" />
+        <exit_code range=":-1" />
+        <regex match="Error:" />
+        <regex match="Exception:" />
+    </stdio>
+    <command>
+<![CDATA[
+    pear
+        #if str( $library.type ) == "paired":
+            -f "$library.forward"
```

Comment:
And we're advocating single quotes for paths. But I guess it's not so straightforward to implement this in planemo, right?